modules/ducktests/tests/ignitetest/services/utils/ignite_aware_app.py

```diff
@@ -82,7 +84,7 @@ def app_args(self):
         args = self.java_class_name + "," + IgniteAwareApplicationService.CONFIG_FILE
 
         if self.params != "":
-            args += "," + self.params
+            args += "," + str(base64.b64encode(json.dumps(self.params).encode("UTF-8")))
```

Comment:
Why do you use base64 there? Is it a workaround to avoid escape issue?

Reply:
Yep

Reply:
Yes

Reply:
Just use single quotes then. This is enough, check how run_tests.sh do the same:
https://github.com/apache/ignite/blob/ignite-ducktape/modules/ducktests/tests/docker/run_tests.sh#L136

Reply:
Max, I suppose that this is more robust solution to problem. Constantly escape everything is a little bit tiresome. base64 solves this problem radically.

Reply:
For example, JSON contains "," already used as a separator at ignite_aware.py because of necessary to transfer some params where only one allowed.

Reply:
ok, understand.